subprojects/docs/src/samples/java/jvm-multi-project-with-code-coverage-distribution/groovy/application/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'myproject.java-conventions'
+    id 'application'
+    id 'jacoco-report-aggregation' // <1>
+}
+
+dependencies {
+    implementation project(':list')
+    implementation project(':utilities')
+}
+
+application {
+    mainClass = 'org.gradle.sample.Main'
+}
+
+// Optional: make JaCoCo aggregate report generation part of the 'check' lifecycle phase
```

Comment:
I like marking this as optional.  is it worth a `<2>` callout?

Reply:
I agree - fixed.